accounts/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+from .models import Profile
```

Comment:
I would add a blank line to separate third-party from local application/library specific imports. Please also do so for the files below.